mypy_boto3_builder/cli_parser.py

```diff
@@ -110,6 +111,11 @@ def parse_args(args: Sequence[str]) -> Namespace:
         action="store_true",
         help="Generate already installed packages for typings folder.",
     )
+    parser.add_argument(
+        "--offline",
+        action="store_false",
+        help="Skip PyPI requests.",
+    )
```

Comment:
Please merge this functionality with `--no-smart-version` flag and update help message for it to reflect that it allows to run builder in offline mode and ignores `--skip-published` flag.